### PR TITLE
ci: update pull-request test failure case

### PR DESCRIPTION
**Pull request by ccalbaugh**
☝️

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>The version was not updated. Please update the version.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>The changelog has not been updated. Please update the changelog.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Branch being merged does not follow [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b507b994706d467dfb43ac0c90533c6607227abd
</p>

File: .github/workflows/pull-request.yaml [hunk 1]
```diff
@@ -25,10 +25,12 @@ jobs:
...
-    - run: npm run lint:ci || true
+    - run: npm run test:ci || touch FAIL_JOB
+    - run: npm run lint:ci || touch FAIL_JOB
     - name: pre-danger
       run: git config user.email "decrapifier@govspend.com" && git config user.name "Decrapifier" && git checkout ${{github.head_ref}} && git config push.default upstream && git branch -u origin/${{github.head_ref}}
     - run: npm run danger
       env:
         DANGER_GITHUB_API_TOKEN: '${{ secrets.PAT_DECRAPIFIER }}'
+    - name: check-if-lint-or-test-runs-failed
+      run: 'if test -f "FAIL_JOB"; then exit 1; fi'
```
Comment (rudolph9) [inline .github/workflows/pull-request.yaml:36]: add back the newline at the end of the file ([explanation](https://unix.stackexchange.com/a/18789))

**Review by rudolph9 — CHANGES_REQUESTED** [review]

File: .github/workflows/pull-request.yaml [hunk 2]
```diff
@@ -33,4 +33,5 @@ jobs:
       env:
         DANGER_GITHUB_API_TOKEN: '${{ secrets.PAT_DECRAPIFIER }}'
     - name: check-if-lint-or-test-runs-failed
-      run: 'if test -f "FAIL_JOB"; then exit 1; fi'
\ No newline at end of file
+      run: 'if test -f "FAIL_JOB"; then exit 1; fi'
+
```
Comment (rudolph9) [inline .github/workflows/pull-request.yaml:37]: Now this line doesn't have a new line 🙃

**Review by rudolph9 — CHANGES_REQUESTED** [review]

File: .github/workflows/pull-request.yaml (hunk 2 above)
Comment (ccalbaugh) [reply .github/workflows/pull-request.yaml:37]: Oh no I overshot it!

**Review by rudolph9 — APPROVED** [review]

**Pull request merged**